src/Pillar-Project/PRProject.class.st

```diff
@@ -134,6 +134,13 @@ PRProject >> defaultOutputDirectoryName [
 	^ Path from: '_result'
 ]
 
+{ #category : #accessing }
+PRProject >> defaultTarget [
+	| defaultExport |
+	defaultExport := (self configuration defaultExport) ifEmpty: [ ^ PRAbsentTarget].
+	(PRTarget associateTarget: defaultExport) ifEmpty: [ ^ PRInvalidTarget ] ifNotEmpty: [ :l | ^ l first ].
```

Comment:
Here they are needed.
But you could replace this method by the `targetByFormatName:ifAbsent:` I was talking above.